graph-proxy/src/graphql/workflows.rs

```diff
@@ -347,4 +356,75 @@ impl WorkflowsQuery {
             .into_result()?;
         Ok(workflow.try_into()?)
     }
+
+    #[instrument(skip(self, ctx))]
+    async fn workflows(
+        &self,
+        ctx: &Context<'_>,
+        proposal_code: String,
+        proposal_number: u32,
+        visit: u32,
+        _continue: Option<String>,
+        limit: Option<u32>,
```

Comment:
We want to follow the [GraphQL cursor connections spec](https://relay.dev/graphql/connections.htm). `async-graphql` has [some tooling](https://async-graphql.github.io/async-graphql/en/cursor_connections.html) for this